trunk/SUAVE/Components/Energy/Networks/Solar.py

```diff
@@ -223,7 +230,7 @@ def evaluate_thrust(self,state):
         
         # link
         battery.inputs = solar_logic.outputs
-        battery.energy_calc(numerics)
+        battery.energy_cycle_model(numerics)
```

Comment:
again, why is this so simple?

Also the solar vehicles could be charging or discharging at any time.....

Reply:
see comment above. keeping old nomenclature